src/routes/CreateSafePage/components/SafeCreationProcess.tsx

```diff
@@ -255,6 +269,28 @@ function SafeCreationProcess(): ReactElement {
           }
         />
       )}
+      {showCouldNotLoadModal && (
+        <GenericModal
+          onClose={onCancel}
+          title="Unable to load the new Safe"
+          body={
+            <div data-testid="safe-could-not-load-popup">
+              <Paragraph>We are currently unable to load the Safe but it was successfully created</Paragraph>
```

Comment:
The test-id here is not necessary because the element is accessible querying by text

Reply:
Is this even being tested?